demos-europe/demosplan-core
# feature (refs T26083): Change how orgaTypes are counted for supportUser statistics

Ticket:
https://yaits.demos-deutschland.de/T26083

Description:
The statistics for support users regarding the Organisation-Type-Counts was retrieved via a method that
asked every user for the organisation and owning roles.
Then used the roles to map them to an orga-tyüe -> incrementing that type-count by one....
The implementation looked only at the first user found for a specific organisation and skipped all others.
This seems wrong as the first user found might be someone that does not have all possible roles within his organisation...

After talking to the PO we decided to simply display the count of every accepted orga-type of interest found for the current customer. Regardless of any user restriction like at least one non deleted user within that organisation....

### How to review/test
login supportUser -> Plattformtools -> Statistik -> Anzahl registrierter Organisationen nach Typ

An Organisation can be of more than one type - so the total amount of types should be equal or greater than the actual Organisation-Count shown in ogin supportUser -> verwalten -> Organisationen.
